tfhe/src/lib.rs

```diff
@@ -2,6 +2,61 @@
 //!
 //! TFHE-rs is a fully homomorphic encryption (FHE) library that implements Zama's variant of TFHE.
 
+// Enable pedantic lints
+#![warn(clippy::pedantic)]
+// Fixing the following lints may be useful
+#![allow(clippy::must_use_candidate)]
```

Comment:
add a comment to indicate that by default those are set to allow and that for now we are introducing lints progressively

Reply:
Changed the comment
Let me know if you have suggestions to improve it

Reply:
just curious what are the comments after each lint ?

Reply:
It's the current number of warnings emitted if we remove the line
This may change with time so it's just to have an idea of the amount of work needed to fix the lint

Reply:
thx